### PR TITLE
feat: Add CalculationOrchestrationState using existing calculation state

### PR DESCRIPTION
**Pull request by lasrinnil**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

**Review by ebbeknudsen — APPROVED** [review]

File: source/dotnet/wholesale-api/WebApi/V3/Calculation/CalculationDto.cs
Comment (dstenroejl) [inline source/dotnet/wholesale-api/WebApi/V3/Calculation/CalculationDto.cs:1]: Observation: If we change this type, should we then generate the "swagger.json" and update the client generated for Subsystem Tests?

**Review by dstenroejl — DISMISSED** [review]
Approved but with an observation

File: source/dotnet/wholesale-api/WebApi/V3/Calculation/CalculationDto.cs
Comment (ebbeknudsen) [reply source/dotnet/wholesale-api/WebApi/V3/Calculation/CalculationDto.cs:1]: Thanks - I've updated the subsystem-test `swagger.json` and the generated client 👍

File: .gitignore [hunk 1]
```diff
@@ -394,3 +394,4 @@ src/streaming/geh_stream.egg-info/
 # Ignore files generated by pytest execution
 test_localsystem.txt
 **/spark-warehouse
+source/dotnet/wholesale-api/WebApi/secrets.json
```
Comment (dstenroejl) [inline .gitignore:397]: Is this on purpose? Usually this kind of files are not located within the source code folder.

**Review by dstenroejl — COMMENTED** [review]
Added a comment I would like to be verified before I can approve.

File: .gitignore (hunk 1 above)
Comment (ebbeknudsen) [reply .gitignore]: I've removed it again 👍

**Review by dstenroejl — APPROVED** [review]
LGTM

**Pull request merged**